### ISSUE TITLE
Add toggle for final dir

### ISSUE THREAD

**Issue by jussipvainionpaa** [madMAx43v3r/chia-plotter#850]
Would it be possible to add a parameter to toggle between 2 different final dirs?
I have lots of 2TB and 3TB drives I'm filling and it would be nice if I could set a toggle between two 2TB drives so I could fill both of them with just one command.

**Comment by diaconom** [comment]
Agreed, I would love this feature, as I create plots on two external drives and would love it to just fill next drive when one is full.

**Comment by Jacek-ghub** [comment]
If you plotting under Windows, then [PSChiaPlotter](https://github.com/MrPig91/PSChiaPlotter) does exactly that.

**Comment by jussipvainionpaa** [comment]
Nah, Ubuntu.

**Comment by Jacek-ghub** [comment]
Then instead of providing those multiple destinations, you can access them through a symlink (given as dest to the plotter), and have your own script manipulate that symlink.

**Comment by jussipvainionpaa** [comment]
I'm sure there's ways to do lots of things already. I just suggested to implement this feature in the plotter itself to make it easy for everyone to use. Seeing as there's already a toggle for temp drives, why not final dirs too?

**Comment by Jacek-ghub** [comment]
I understand what you said, and partially agree with you.  

However, my take is that a plotter should focus on plotting, and plot manager on plot management.  Just to provide more context on what you requested, would you want to do the toggle you described in a round-robin way (every new plot cycles through the list of destination drives), or rather serialized way (fill the first destination first, then go to the next, etc.).  Each of those may not share much code.

So, just looking at either serializing or parallelizing those destinations creates more complications for the plotter (I would prefer serialization, as that would give me plenty of time to remove those full drives, and add new ones waiting for their turn, where having all the drives with the same capacity would end up with all drives full in the middle of the night, as such plotter going idle).

Again, I would really love to see speed improvements in the plotter every day.  I would also like to see a plot manager that would improve with time, providing features like you have just requested.  The coding requirements may not be the same for those two tasks, so why put the extra burden on the plotter side.

**Comment by jussipvainionpaa** [comment]
I see your point regarding plot creating and plot management. In the end it would be best if one software would combine all of the features so we wouldn't need to install and configure bunch of different ones and hope that every independent developer keeps actively working on the software.

What I was looking for here is a round robin type of toggle (like the one with temp drives) but I could see it being useful too if we could provide multiple final dirs that would be used one by one after the first is full.

Actually it would be better to implement the latter as it would also work for my situation. And the round robin type would indeed require same size drives.

**Comment by Jacek-ghub** [comment]
The bottom line is that it is much better for the community to get just one/two tools be done by a handful of people, than all of us screwing around with half-ass solutions.  So, voicing opinions like yours is definitely a plus, and potentially sharing/improving those scripts benefits a bunch of people.

I am really happy with that PSChiaPlotter manager (sorry, Win only), as it is really an extremely well done piece of software (the best manager, IMO).  Yes, it has kinks, but it beats hands down the official Chia UI.  The same can be said about this MM plotter.  So, I am waiting here for plotting times going under one minute :)

**Comment by programsalon** [comment]
you can write multi commands, or write a program to remove plot file to another disk auto.

**Comment by wallentx** [comment]
I've found nothing better, and will be including logic to replot soon.
https://gist.github.com/wallentx/a462521b0c8ea0ebd64d19f0c1ed82f9
